cg/meta/workflow/nf_analysis.py

```diff
@@ -307,3 +314,85 @@ def get_deliverables_for_case(self, case_id: str) -> PipelineDeliverables:
                 )
             files.append(FileDeliverable(**file))
         return PipelineDeliverables(files=files)
+
+    def get_multiqc_json_path(self, case_id: str) -> Path:
+        """Return the path of the multiqc_data.json file."""
+        return Path(
+            self.root_dir,
+            case_id,
+            MultiQC.MULTIQC,
+            MultiQC.MULTIQC_DATA,
+            MultiQC.MULTIQC_DATA + FileExtensions.JSON,
+        )
+
+    def get_pipeline_metrics(self) -> dict:
+        """Get nf-core pipeline metrics constants."""
+        return {}
```

Comment:
The purpose of this function is to return the constants metric conditions for rnafusion and taxprofiler to begin with. In `cg/meta/workflow/rnafusion.py`, there is this one:

   ```
 def get_pipeline_metrics(self) -> dict:
        return RNAFUSION_METRIC_CONDITIONS
```

Since for taxprofiler we have not agreed on any constant metrics yet, the idea was to add this function here and return an empty dictionary.

Reply:
Ohh ok I understand now 👍 I will resolve this